### PR TITLE
Added the correct PACKETVER Date for MSG_NPC_WORK_IN_PROGRESS message.

### PR DESCRIPTION
**Pull request by Jedzkie**
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Pull Request Prelude

[//]: # (Thank you for working on improving Hercules!)

[//]: # (Please complete these steps and check the following boxes by putting an `x` inside the brackets _before_ filing your Pull Request.)

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

[//]: # (Describe at length, the changes that this pull request makes.)

**Affected Branches:** master

[//]: # (Master? Slave?)

**Issues addressed:**

[//]: # (Issue Tracker Number if any.)

### Known Issues and TODO List
The message string only shows if mode is RENEWAL only.

- [x] enable message for <= 20110309 clients.

[//]: # (Insert checklist here)
[//]: # (Syntax: - [ ] Checkbox)

[//]: # (**NOTE** Enable the setting "[√] Allow edits from maintainers." when creating your pull request if you have not already enabled it.)

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by HerculesWSAPI** [comment]
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1694)

File: src/map/clif.c [hunk 1]
```diff
@@ -10632,9 +10632,9 @@ void clif_parse_NpcClicked(int fd,struct map_session_data *sd)
 		clif->clearunit_area(&sd->bl,CLR_DEAD);
 		return;
 	}
-	if( sd->npc_id || sd->state.workinprogress&2 ){
-#ifdef RENEWAL
-		clif->msgtable(sd, MSG_NPC_WORK_IN_PROGRESS); // TODO look for the client date that has this message.
+	if (sd->npc_id || sd->state.workinprogress&2) {
```
Comment (Smokexyz) [inline src/map/clif.c]: `sd->state.workinprogress & 2`

File: src/map/clif.c [hunk 2]
```diff
@@ -11032,12 +11032,12 @@ void clif_parse_ChangeCart(int fd,struct map_session_data *sd)
 	if( pc->checkskill(sd, MC_CHANGECART) < 1 )
 		return;
 
-#ifdef RENEWAL
-	if( sd->npc_id || sd->state.workinprogress&1 ){
+	if (sd->npc_id || sd->state.workinprogress&1) {
```
Comment (Smokexyz) [inline src/map/clif.c]: `sd->state.workinprogress & 1`

File: src/map/clif.c [hunk 3]
```diff
@@ -11244,9 +11244,9 @@ void clif_parse_UseSkillToId(int fd, struct map_session_data *sd)
 	// Whether skill fails or not is irrelevant, the char ain't idle. [Skotlex]
 	pc->update_idle_time(sd, BCIDLE_USESKILLTOID);
 
-	if( sd->npc_id || sd->state.workinprogress&1 ){
-#ifdef RENEWAL
-		clif->msgtable(sd, MSG_NPC_WORK_IN_PROGRESS); // TODO look for the client date that has this message.
+	if (sd->npc_id || sd->state.workinprogress&1) {
```
Comment (Smokexyz) [inline src/map/clif.c]: `sd->state.workinprogress & 1`

File: src/map/clif.c [hunk 4]
```diff
@@ -11341,12 +11341,12 @@ void clif_parse_UseSkillToPosSub(int fd, struct map_session_data *sd, uint16 ski
 		return;
 	}
 
-#ifdef RENEWAL
-	if( sd->state.workinprogress&1 ){
-		clif->msgtable(sd, MSG_NPC_WORK_IN_PROGRESS); // TODO look for the client date that has this message.
+	if (sd->state.workinprogress&1) {
```
Comment (Smokexyz) [inline src/map/clif.c]: `sd->state.workinprogress & 1`

File: src/map/pc.c [hunk 5]
```diff
@@ -5019,10 +5019,9 @@ int pc_useitem(struct map_session_data *sd,int n) {
 	nullpo_ret(sd);
 	Assert_ret(n >= 0 && n < MAX_INVENTORY);
 
-	if( sd->npc_id || sd->state.workinprogress&1 ){
-		/* TODO: add to clif->messages enum */
-#ifdef RENEWAL
-		clif->msgtable(sd, MSG_NPC_WORK_IN_PROGRESS); // TODO look for the client date that has this message.
+	if (sd->npc_id || sd->state.workinprogress&1) {
```
Comment (Smokexyz) [inline src/map/pc.c]: `sd->state.workinprogress & 1`

File: src/map/clif.c (hunk 1 above)
Comment (Jedzkie) [inline src/map/clif.c]: Done.

File: src/map/clif.c (hunk 2 above)
Comment (Jedzkie) [inline src/map/clif.c]: Done.

File: src/map/clif.c (hunk 3 above)
Comment (Jedzkie) [inline src/map/clif.c]: Done.

File: src/map/clif.c (hunk 4 above)
Comment (Jedzkie) [inline src/map/clif.c]: Done.

File: src/map/pc.c (hunk 5 above)
Comment (Jedzkie) [inline src/map/pc.c]: Done.

**Comment by Jedzkie** [comment]
@MishimaHaruna can we merge this now?

**Comment by MishimaHaruna** [comment]
Merged, thanks!

**Comment by tlacson7** [comment]
this breaks on PreRE using 20100730 

```
pc.c: In function ‘pc_useitem’:
pc.c:5042:27: error: ‘fd’ undeclared (first use in this function)
   clif->messagecolor_self(fd, COLOR_WHITE, msg_fd(fd, 48));
                           ^
pc.c:5042:27: note: each undeclared identifier is reported only once for each function it appears in
make[1]: *** [obj_sql/pc.o] Error 1

```

File: src/map/clif.c [hunk 6]
```diff
@@ -10632,9 +10632,11 @@ void clif_parse_NpcClicked(int fd,struct map_session_data *sd)
...
 		return;
 	}
-	if( sd->npc_id || sd->state.workinprogress&2 ){
-#ifdef RENEWAL
-		clif->msgtable(sd, MSG_NPC_WORK_IN_PROGRESS); // TODO look for the client date that has this message.
+	if (sd->npc_id || sd->state.workinprogress & 2) {
+#if PACKETVER >= 20110309
+		clif->msgtable(sd, MSG_NPC_WORK_IN_PROGRESS);
+#else
+		clif->messagecolor_self(fd, COLOR_WHITE, msg_fd(fd, 48));
```
Comment (dastgir) [inline src/map/clif.c]: As for mentioned bug, this should be sd->fd

File: src/map/pc.c [hunk 7]
```diff
@@ -5019,10 +5019,11 @@ int pc_useitem(struct map_session_data *sd,int n) {
...
 
-	if( sd->npc_id || sd->state.workinprogress&1 ){
-		/* TODO: add to clif->messages enum */
-#ifdef RENEWAL
-		clif->msgtable(sd, MSG_NPC_WORK_IN_PROGRESS); // TODO look for the client date that has this message.
+	if (sd->npc_id || sd->state.workinprogress & 1) {
+#if PACKETVER >= 20110309
+		clif->msgtable(sd, MSG_NPC_WORK_IN_PROGRESS);
+#else
+		clif->messagecolor_self(fd, COLOR_WHITE, msg_fd(fd, 48));
```
Comment (dastgir) [inline src/map/pc.c]: Should be sd->fd (related to #1721)

**Pull request merged**